### ISSUE TITLE
Learning rate and scheduler for stage 1 training.

### ISSUE THREAD

**Issue by duchenzhuang** [thuanz123/enhancing-transformers#21]
Hi, @thuanz123 

Thank you for reproducing and open-sourcing ViT-VQGAN using PyTorch. I would like to inquire, is the learning rate fixed at 4.5e-6 during stage 1 training in your code? I noticed that the paper mentions a base_lr of 1e-4, along with linear warm-up and cosine decay.

**Comment by donthomasitos** [comment]
I tried the scheduler from the paper, turns out to be totally instable. I'd also be curious to know the reason. The architecture seems identical.
